### PR TITLE
satunivers.tv

### PR DESCRIPTION
**Pull request by wout-codes**
server.satunivers.tv #73164

### CODE REVIEW

**Comment by wout-codes** [comment]
@zloyden Check this one first before I make a new PR.

**Review by zloyden — CHANGES_REQUESTED** [review]
`server.satunivers.tv` -> just `satunivers.tv`

**Comment by wout-codes** [comment]
> `server.satunivers.tv` -> just `satunivers.tv`

Should I use an Asterisk (*). There might be more domains like this.

**Comment by zloyden** [comment]
> > `server.satunivers.tv` -> just `satunivers.tv`
> 
> Should I use an Asterisk (*). There might be more domains like this.

`satunivers.tv` is enough for now.

File: EnglishFilter/sections/content_blocker.txt [hunk 1]
```diff
@@ -85,6 +85,7 @@ cdn.braun634.com^$third-party
 @@||zoover.adtrackers.net/shared/bannerpages/darttagsbanner.aspx
 @@||zoover.bannertrack.net/shared/bannerpages/darttagsbanner.aspx
 !
+@@||satunivers.tv^$generichide
```
Comment (zloyden) [inline EnglishFilter/sections/content_blocker.txt:88]: Could you move it a little lower? `!--- antiblock.org g207---` is a right place.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Comment by wout-codes** [comment]
@zloyden Done. Please check.

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
> @zloyden Done. Please check.

I removed a duplicate, it's ok now. Thanks)

**Pull request merged**